model/project_ref.go

```diff
@@ -1896,27 +1904,24 @@ func SaveProjectPageForSection(projectId string, p *ProjectRef, section ProjectP
 		if p.TracksPushEvents != nil {
 			setUpdate[ProjectRefTracksPushEventsKey] = p.TracksPushEvents
 		}
+		config, error := evergreen.GetConfig()
```

Comment:
shouldn't use error since this is the package name. If you want to have distinct error names I would make this one specific, like configErr. It's pretty unclear to me what's going on here though.